cpp/src/plasma/test/client_tests.cc

```diff
@@ -386,7 +384,6 @@ TEST_F(TestPlasmaStore, AbortTest) {
   // Test that we can get the object.
   ARROW_CHECK_OK(client_.Get({object_id}, -1, &object_buffers));
   AssertObjectBufferEqual(object_buffers[0], {42, 43}, {1, 2, 3, 4, 5});
```

Comment:
The following line was a bug masked by the release cache. This form of get calls Release in the destructor of the PlasmaBuffer shared pointer.